pandas/core/frame.py

```diff
@@ -2865,16 +2865,29 @@ def to_html(
         justify: str | None = None,
         max_rows: int | None = None,
         max_cols: int | None = None,
-        show_dimensions: bool | str = False,
+        show_dimensions: bool | str | None = None,
         decimal: str = ".",
-        bold_rows: bool = True,
+        bold_rows: bool | None = None,
         classes: str | list | tuple | None = None,
         escape: bool = True,
-        notebook: bool = False,
+        notebook: bool | None = None,
         border: int | None = None,
         table_id: str | None = None,
-        render_links: bool = False,
+        render_links: bool | None = None,
         encoding: str | None = None,
+        *,
+        table_attributes: str | None = None,
```

Comment:
why are you adding the *new* options?

Reply:
this suggestion by @jorisvandenbossche https://github.com/pandas-dev/pandas/pull/44451#issuecomment-1000741346